mne/tests/test_surface.py

```diff
@@ -244,4 +252,4 @@ def test_normal_orth():
         assert_allclose(ori[2], nn, atol=1e-12)
 
 
-run_tests_if_main()
+#run_tests_if_main()
```

Comment:
remove

Reply:
BTW now that we have `pytest` and it's (much) easier to use, I find these not so helpful. I'd be okay with getting rid of them in the codebase...

Reply:
Alternatively, is there a good way to run a single test inside a Jupyter console? (I usually need the postmortem debugger)